src/main/resources/db/changelog/create/0004_approved_proxy_application.sql

```diff
@@ -0,0 +1,44 @@
+-- liquibase formatted sql
+-- changeset Krister.Bone@valtech.com:create_approved_proxy_application_table endDelimiter:;
+
+DROP TABLE IF EXISTS `approved_proxy_application`;
+CREATE TABLE `approved_proxy_application`
+(
+    `application_id`            varchar(24)  NOT NULL,
+    `created_at`                timestamp    NOT NULL,
+    `gss_code`                  varchar(9)   NOT NULL,
+    `source`                    varchar(50)  NOT NULL,
+    `authorised_at`             timestamp    NOT NULL,
+    `authorising_staff_id`      varchar(255) NOT NULL,
+    `first_name`                varchar(35)  NOT NULL,
+    `middle_names`              varchar(100),
+    `surname`                   varchar(35)  NOT NULL,
+    `dob`                       date,
+    `email`                     varchar(255),
+    `phone`                     varchar(50),
+    `registered_address_id`     char(36)     NOT NULL,
+    `reference_number`          varchar(10)  NOT NULL,
+    `ip_address`                varchar(45)  NOT NULL,
+    `ems_elector_id`            varchar(255) NOT NULL,
+    `language`                  char(2),
+    `proxy_first_name`          varchar(35)  NOT NULL,
+    `proxy_middle_names`        varchar(100),
+    `proxy_surname`             varchar(35)  NOT NULL,
+    `proxy_family_relationship` varchar(500),
+    `proxy_address_id`          char(36),
```

Comment:
Is proxy address mandatory and so should this be not null?

Reply:
Done